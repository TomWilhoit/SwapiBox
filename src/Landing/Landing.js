import React, { Component } from "react";
import "../Landing/Landing.scss";

class Landing extends Component {
  render() {
    return (
      <div className="landing">
        <div className="fade">
          <section className="star-wars">
            <div className="crawl">
              <div className="title">
                <p>Episode {this.props.scrollEp}</p>
                <h1>{this.props.scrollTitle}</h1>
              </div>
              {this.props.scrollText}
            </div>
          </section>
        </div>
      </div>
    );
  }
}

export default Landing;
