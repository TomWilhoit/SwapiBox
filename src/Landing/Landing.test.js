import React from "react";
import Landing from "../Landing/Landing";
import { shallow } from "enzyme";

describe("People", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Landing />);
  });

  it("should match snapshot when all data is passed correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
