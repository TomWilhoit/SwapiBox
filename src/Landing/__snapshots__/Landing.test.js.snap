// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`People should match snapshot when all data is passed correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Landing />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="fade"
      >
        <section
          className="star-wars"
        >
          <div
            className="crawl"
          >
            <div
              className="title"
            >
              <p>
                Episode 
              </p>
              <h1 />
            </div>
          </div>
        </section>
      </div>,
      "className": "landing",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <section
          className="star-wars"
        >
          <div
            className="crawl"
          >
            <div
              className="title"
            >
              <p>
                Episode 
              </p>
              <h1 />
            </div>
          </div>
        </section>,
        "className": "fade",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="crawl"
          >
            <div
              className="title"
            >
              <p>
                Episode 
              </p>
              <h1 />
            </div>
          </div>,
          "className": "star-wars",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="title"
              >
                <p>
                  Episode 
                </p>
                <h1 />
              </div>,
              undefined,
            ],
            "className": "crawl",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <p>
                    Episode 
                  </p>,
                  <h1 />,
                ],
                "className": "title",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      "Episode ",
                      undefined,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    "Episode ",
                    undefined,
                  ],
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "h1",
                },
              ],
              "type": "div",
            },
            undefined,
          ],
          "type": "div",
        },
        "type": "section",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="fade"
        >
          <section
            className="star-wars"
          >
            <div
              className="crawl"
            >
              <div
                className="title"
              >
                <p>
                  Episode 
                </p>
                <h1 />
              </div>
            </div>
          </section>
        </div>,
        "className": "landing",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <section
            className="star-wars"
          >
            <div
              className="crawl"
            >
              <div
                className="title"
              >
                <p>
                  Episode 
                </p>
                <h1 />
              </div>
            </div>
          </section>,
          "className": "fade",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="crawl"
            >
              <div
                className="title"
              >
                <p>
                  Episode 
                </p>
                <h1 />
              </div>
            </div>,
            "className": "star-wars",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="title"
                >
                  <p>
                    Episode 
                  </p>
                  <h1 />
                </div>,
                undefined,
              ],
              "className": "crawl",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <p>
                      Episode 
                    </p>,
                    <h1 />,
                  ],
                  "className": "title",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        "Episode ",
                        undefined,
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      "Episode ",
                      undefined,
                    ],
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": undefined,
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "h1",
                  },
                ],
                "type": "div",
              },
              undefined,
            ],
            "type": "div",
          },
          "type": "section",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
